iphone/Classes/TiAppiOSUserNotificationActionProxy.m

```diff
@@ -79,38 +75,22 @@ - (id)notificationAction
 
 - (void)setActivationMode:(id)value
 {
-  if ([TiUtils isIOSVersionOrGreater:@"10.0"]) {
-    return; // Not available on iOS 10+
-  }
-
-  [(UIMutableUserNotificationAction *)[self notificationAction] setActivationMode:[TiUtils intValue:value]];
+  DebugLog(@"[WARN] This property is not available in iOS 10+.");
 }
 
 - (void)setBehavior:(id)value
 {
-  if ([TiUtils isIOSVersionOrGreater:@"10.0"]) {
-    return; // Not available on iOS 10+
-  }
-
-  [(UIMutableUserNotificationAction *)[self notificationAction] setBehavior:[TiUtils intValue:value]];
+  DebugLog(@"[WARN] This property is not available in iOS 10+.");
 }
 
 - (void)setDestructive:(id)value
 {
-  if ([TiUtils isIOSVersionOrGreater:@"10.0"]) {
-    return; // Not available on iOS 10+
-  }
-
-  [(UIMutableUserNotificationAction *)[self notificationAction] setDestructive:[TiUtils boolValue:value]];
+  DebugLog(@"[WARN] This property is not available in iOS 10+.");
```

Comment:
mark as deprecated/removed in 9.0.0 in our docs

Reply:
Actually these properties are available but at creation time only for iOS 10+. Manual setters are not available. I think I should delete these setters.

Reply:
@sgtcoolguy Deleted setters as these are creation-only. These properties are already used while calling - (void)_initWithProperties:(NSDictionary *)properties.